orchagent/notifications.cpp

```diff
@@ -17,15 +17,16 @@ void on_port_state_change(uint32_t count, sai_port_oper_status_notification_t *d
 
     lock_guard<mutex> lock(gDbMutex);
 
-    /* Wait until gPortsOrch is initialized */
-    if (!gPortsOrch || !gPortsOrch->isInitDone())
+    if (!gPortsOrch)
```

Comment:
missing notifications at this stage is acceptable. when the gportsorch pointer is not there, it means that the orchagent does not even starts. during regular orchagent starts, no ports are created before gportsorch pointer is initialized.